src/cargo/ops/cargo_run.rs

```diff
@@ -26,7 +26,7 @@ pub fn run(
         },
     };
 
-    let bins: Vec<_> = pkg.manifest()
+let bins: Vec<_> = pkg.manifest()
```

Comment:
Indentation is off. Might be a good idea to run `rustfmt +stable` over the changed files. Note that we don't do that systematically for various reasons though, so that might touch some unrelated code unfortunately.